index.html

```diff
@@ -2,26 +2,26 @@
 <html lang="en">
 <head>
     <!-- Favicons -->
-    <link rel="apple-touch-icon" sizes="57x57" href="/images/favicon.ico/apple-icon-57x57.png">
-    <link rel="apple-touch-icon" sizes="60x60" href="/images/favicon.ico/apple-icon-60x60.png">
-    <link rel="apple-touch-icon" sizes="72x72" href="/images/favicon.ico/apple-icon-72x72.png">
-    <link rel="apple-touch-icon" sizes="76x76" href="/images/favicon.ico/apple-icon-76x76.png">
-    <link rel="apple-touch-icon" sizes="114x114" href="/images/favicon.ico/apple-icon-114x114.png">
-    <link rel="apple-touch-icon" sizes="120x120" href="/images/favicon.ico/apple-icon-120x120.png">
-    <link rel="apple-touch-icon" sizes="144x144" href="/images/favicon.ico/apple-icon-144x144.png">
-    <link rel="apple-touch-icon" sizes="152x152" href="/images/favicon.ico/apple-icon-152x152.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/images/favicon.ico/apple-icon-180x180.png">
-    <link rel="icon" type="image/png" sizes="192x192"  href="/images/favicon.ico/android-icon-192x192.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon.ico/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="96x96" href="/images/favicon.ico/favicon-96x96.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon.ico/favicon-16x16.png">
-    <link rel="manifest" href="/images/favicon.ico/manifest.json">
+    <link rel="apple-touch-icon" sizes="57x57" href="images/favicon.ico/apple-icon-57x57.png">
```

Comment:
He’s using relative paths because absolute paths don’t work on local, since the file isn’t been rendered from a server.

Reply:
@Demilad Well, absolute paths are best practice, especially for SEO and page hyperlinking. 
However, there are numerous ways to handle absolute paths from front-end or back-end. 
This method uses a forward slash "/" which implies the root or domain of the server; e.g. for a windows local machine, the root in-turn is `file:///C:/` so a path like:
`href="/images/favicon.ico/apple-icon-57x57.png"`
is interpreted as 
**LOCAL MACHINE**(windows): `href="file:///C:/images/favicon.ico/apple-icon-57x57.png"`
**SERVER**(no matter the sub-folder): `href="example.com/images/favicon.ico/apple-icon-57x57.png"`

Relative paths is not all that bad especially in a single page site like this one. And since it works on both server and local machine hence my reason for using it.

**SUMMARY:** _Absolute links/paths are not appropriate in many circumstances. The general rule is to use relative links/paths when at all possible._